app/main/views/briefs.py

```diff
@@ -162,21 +162,27 @@ def edit_brief_response(brief_id, brief_response_id, section_id=None):
         brief['frameworkSlug'], 'edit_brief_response'
     ).filter({'lot': lot['slug'], 'brief': brief, 'max_day_rate': max_day_rate})
 
-    if section_id is None:
+    section = content.get_section(content.get_next_editable_section_id())
+    if section is None or not section.editable:
```

Comment:
If a section isn't editable do we definitely want to abort? What if in the future there was a brief type that could have a none editable section in it (for information maybe as you went through the flow)?
I might be talking rubbish and that may never happen though...this is just speculation...